cpp/src_prims/distance/fused_l2_nn.cuh

```diff
@@ -254,6 +274,35 @@ struct FusedL2NN : public BaseClass {
       }
     }
   }
+
+#if (ENABLE_MEMCPY_ASYNC == 1)
+  ///@todo: fix this to use memcpy_async
+  DI void ldgXY(IdxT kidx) {
```

Comment:
Could you add a comment explaining this function? Wasn't 100% clear from the naming.

Reply:
Actually, these methods are overriding the corresponding ones in the base class. I have documented these methods and their purposes there.